package.json

```diff
@@ -19,7 +19,6 @@
     "approximate-number": "^2.0.0",
```

Comment:
I wasn't 100% sure of what's exactly incorrect in all that: the fact that these icons are dropped on the new line on a wider viewport than it used to be before, or that there're huge spaces between the icons. So I assumed the last one and made them to be `space-around` instead of `space-between` :worried: